internal/spec/spec.go

```diff
@@ -114,29 +108,24 @@ func ParamsToPodLabels(controllerID string, bootstrapParams params.BootstrapInst
 	return labels
 }
 
-func FlavourToResourceRequirements(flavour Flavour) corev1.ResourceRequirements {
-	resourceCPU := "500m"
-	resourceMemory := "500Mi"
-
-	switch flavour {
-	case Medium:
-		resourceCPU = "1000m"
-		resourceMemory = "1Gi"
-	case Large:
-		resourceCPU = "2000m"
-		resourceMemory = "2Gi"
+func FlavourToResourceRequirements(flavour string) corev1.ResourceRequirements {
+	if _, ok := config.Config.Flavours[flavour]; !ok {
+		resourceCPURequest := "100m"
+		resourceMemoryRequest := "100Mi"
+		resourceMemoryLimit := "200Mi"
```

Comment:
do we really want to use a default flavour if the specified flavour is not configured or would it be better to return an error?

wdyt @bavarianbidi?

Reply:
IMO we should provide a default, as it is another hurdle for the enduser debugging this and finding the error if there is no default.

Reply:
other providers don't do a default but from a user perspective a default might be fine. Unfortunately it's not that easy to return an info to the user that the default was used.